contracts/ExtraRewardStash.sol

```diff
@@ -127,25 +127,23 @@ contract ExtraRewardStash is IStash {
     function setToken(address _token) internal {
         TokenInfo storage t = tokenInfo[_token];
 
-        if (t.token == address(0)) {
```

Comment:
Remove this IF statement because it blocked the following scenario:
1. Add multiple tokens as a rewards
2. Clear rewards
3. Adding back one of those two tokens would not work because this if wouldn't be address(0)

Reply:
Maybe we should clear them during the removal? Or it would be too complicated?